gridt/src/app/home/home.page.html

```diff
@@ -72,7 +72,7 @@ <h3>
                         </ion-text>
                           <app-expandable [expandedHeight]="itemExpandedHeight" [expanded]="itemExpanded">
                             <ion-text *ngIf="itemExpanded">
-                              <p class="overflow-visible" size="auto">{{ leader.last_signal.message }}</p>
+                              <p class="overflow-visible">{{ leader.last_signal.message }}</p>
```

Comment:
I personally really liked the idea of having to tap on the message to see it. That would mean making this a conditional statement.

Reply:
That should not be there as it does nothing.
And i agree with you, that is what it does already!